dev-docs/source/environment.rst

```diff
@@ -60,6 +78,23 @@ Remote API Endpoints
     HC_CERTFILE = ...
 
 
+Cloud environment
+-----------------
+
+Depending on where the DSO API is being run, it might need to perform additional configuration.
+
+We support Azure logging and tracing when the ``CLOUD_ENV=azure`` setting is used.
+When running in Azure two additional environment variables need to be
+specified:
+
+-  ``AZURE_API_INSTRUMENTATION_KEY``
+-  ``AZURE_API_INGESTION_ENDPOINT``
```

Comment:
Current `README.md` on `master` specifies them to be:

- `AZURE_APPI_INSTRUMENTATION_KEY`
- `AZURE_APPI_INGESTION_ENDPOINT`

Have you used the latest `master`?

Reply:
Or did you think `APPI` was a spelling error? It is not. See commit message for fd41d1799425c88b2ea918e5b017421901a211b6

Reply:
Whoops.. indeed I thought it was a spelling error. I've added the link in the documentation as well.